tools/internal_ci/linux/grpc_e2e_performance_v2.sh

```diff
@@ -53,12 +53,6 @@ else
 fi
 GRPC_GO_GITREF="$(git ls-remote https://github.com/grpc/grpc-go.git master | cut -f1)"
 GRPC_JAVA_GITREF="$(git ls-remote https://github.com/grpc/grpc-java.git master | cut -f1)"
-# Prebuilt driver comes from the latest test-infra release.
-LATEST_TEST_INFRA_RELEASE="$(curl -s https://api.github.com/repos/grpc/test-infra/releases | jq '.[0].tag_name' | tr -d '"')"
```

Comment:
it's nice that we can remove this, but where is the driver image name taken from then?

Reply:
See above, it comes from the default driver image built with the test infrastructure release. This image uses a driver binary taken from a gRPC release specified at build time.

Reply:
In practical terms, the driver image will be the same. We specified where to get it from (recently) because of a bug, where it was not possible to specify the name of the driver pod without also specifying the image. This change removes the limitation, so we stop specifying the image location (which is inferred by the infrastructure and will typically point to the same one).